Requirements windows

KC2016 opened the pull request:
insert a file of requirements to an environment using windows (windows10)

KC2016 committed:
Include installation to windows

install also *pywin32*, and instability of the *pyowm*

KC2016 committed:
Revert "Include installation to windows"

This reverts commit f8ac626dd28645d17e598121fa057b130c1db55c.

KC2016 committed:
Insert installation in windows

install pywin32 and instability of pyowm

KC2016 committed:
requirements to windows

insert a file of requirements to an environment using windows (windows10)

satyammittal commented on README.md:
````diff
@@ -81,11 +81,14 @@ You can quit by pressing <kbd>Ctrl</kbd>+<kbd>C</kbd>
     ```brew install portaudio``` (mac) <br>
     - Further, install other requirements using : <br>
-    ```pip install -r requirements.txt```
+    ```pip install -r requirements.txt``` <br>
````
Please specify here if using windows, use ```pip install -r requirements_windows.txt```

KC2016 committed:
add install requirements to windows in readme

add install requirements to windows in readme and say possible instability of pyowm in this system
